### DISCUSSION TITLE
Import "httpx" could not be resolved Pylance(reportMissingImports)

### DISCUSSION THREAD

**Discussion by phoenixor** [encode/httpx#2110] [Q&A]
i use windows 10, vscode1.65.0, python 3.10.1
the code runs correctly, but the import got an error from Pylance:
![image](https://user-images.githubusercontent.com/6568115/156866650-95952ba0-3810-4993-a473-ec5d89d514d8.png)

**Comment by lovelydinosaur** [comment]
No idea. You probably need to ask the pylance community about this.
